internal/evaluation_target/definition/definition.go

```diff
@@ -18,35 +18,22 @@ package definition
 
 import (
 	"context"
-	"fmt"
-
-	"github.com/spf13/afero"
 
 	"github.com/hacbs-contract/ec-cli/internal/evaluator"
 	"github.com/hacbs-contract/ec-cli/internal/policy"
 	"github.com/hacbs-contract/ec-cli/internal/policy/source"
-	"github.com/hacbs-contract/ec-cli/internal/utils"
 )
 
-var newConftestEvaluator = evaluator.NewConftestEvaluator
-var pathExists = afero.Exists
+var newConftestEvaluator = evaluator.NewConftestEvaluatorWithNamespace
 
 // DefinitionFile represents the structure needed to evaluate a pipeline definition file
 type Definition struct {
-	Fpath     string
+	Fpath     []string
 	Evaluator evaluator.Evaluator
 }
 
 // NewPipelineDefinitionFile returns a DefinitionFile struct with FPath and evaluator ready to use
-func NewDefinition(ctx context.Context, fpath string, sources []source.PolicySource) (*Definition, error) {
-	fs := utils.FS(ctx)
-	exists, err := pathExists(fs, fpath)
-	if err != nil {
-		return nil, err
-	}
-	if !exists {
-		return nil, fmt.Errorf("fpath '%s' does not exist", fpath)
-	}
+func NewDefinition(ctx context.Context, fpath []string, sources []source.PolicySource, namespace []string) (*Definition, error) {
```

Comment:
It does do that. I can add a test though. The thing is, since we include the policy/lib directory, conftest evaluates the definition file against all namespaces under policy/. So you end up with this. On one hand it's great because you can see all failed and successful policies/packages, but you also see conftest runs that don't matter. To address this, I can merge all the namespaces and drop the `namespace` field or we can require the `namespace` param, but support something like `policy.pipeline` which includes everything under pipeline. 

```yaml
[
 {
    "filename": "../build-definitions/task/buildah/0.1/buildah.yaml",
    "violations": [],
    "warnings": [],
    "success": true,
    "namespace": "lib.image"
  },
  {
    "filename": "../build-definitions/task/buildah/0.1/buildah.yaml",
    "violations": [
      {
        "msg": "Unexpected kind 'Task'",
        "metadata": {
          "code": "basic.unexpected_kind",
          "description": "A sanity check to confirm the input data has the kind \"Pipeline\"",
          "effective_on": "2022-01-01T00:00:00Z",
          "title": "Input data has unexpected kind"
        }
      }
    ],
    "warnings": [],
    "success": false,
    "namespace": "policy.pipeline.basic"
  }
]

Reply:
I'll add. I think allowing conftest to run against `lib` feels like the wrong thing.

Reply:
`validate image` runs against all namespaces and we had to do something special to handle this. Check it out to see how it was addressed.

Reply:
the report output has been updated. https://github.com/hacbs-contract/ec-cli/pull/502/commits/0b56973ec2c02d3a93394915d9c7a476c21ba588